Client/Frontend/Browser/SwipeAnimator.swift

```diff
@@ -45,9 +47,14 @@ class SwipeAnimator: NSObject {
 
         super.init()
 
-        let panGesture = UIPanGestureRecognizer(target: self, action: #selector(SwipeAnimator.SELdidPan(_:)))
-        container.addGestureRecognizer(panGesture)
-        panGesture.delegate = self
+        self.panGestureRecogniser = UIPanGestureRecognizer(target: self, action: #selector(SwipeAnimator.SELdidPan(_:)))
+        container.addGestureRecognizer(self.panGestureRecogniser)
+        self.panGestureRecogniser.delegate = self
+    }
+
+    func cancelExistingGestures() {
+        self.panGestureRecogniser.enabled = false
+        self.panGestureRecogniser.enabled = true
```

Comment:
Heh, we also do this in `ContextMenuHelper.swift`. Mildly tempted to create an extension that adds `cancel()`, though probably not worth the effort for just 2 lines.